packages/bits/demo/src/components/demo/wizard-v2/wizard-dynamic/wizard-dynamic.example.component.ts

```diff
@@ -1,6 +1,6 @@
 import {AfterViewInit, Component, TemplateRef, ViewChild} from "@angular/core";
 
-export interface IWizardStepConfig {
+export interface IWizardStepV2Component {
```

Comment:
What's the reason for this rename? To me this name sounds like a nova library interface instead of a demo consumer-side interface.

Reply:
Would it make sense to move it to the library? If so, I think `IWizardStepConfig` works as a name. If we keep it local to the examples, maybe just `IStepConfig` since the `Wizard` part is kind of redundant in this context.

Reply:
This is a consumer-defined interface, its only purpose is to make it easier to work with the code and make typescript happy. This particular interface name explicitly describes the configuration target. Consumers can certainly use a different name or decide not to use any interfaces at all. I don't think we should restrict them with any pre-defined interfaces here.

Reply:
I renamed the interface to **IWizardStepData**. Hope this sounds better now

Reply:
that does sound better :)